src/CakeSlayer/Monad.hs

```diff
@@ -63,6 +76,11 @@ instance (Show err, Typeable err)
         ioAction `catch` \(AppException e) -> runApp env $ handler e
     {-# INLINE catchError #-}
 
+instance MonadMonitor (App err env) where
```

Comment:
Should we add some docs? Like why it's needed?

Reply:
I can add some docs, they never harm.